sdk/cognitiveservices/azure-cognitiveservices-language-luis/CHANGELOG.md

```diff
@@ -68,7 +68,7 @@
   - Model FeatureInfoObject has a new parameter
     enabled_for_all_models
   - Model LuisApp has a new parameter phraselists
-  - Model LuisApp has a new parameter hierarchicals
+  - Model LuisApp has a new parameter hierarchicals <!-- cspell: disable-line -->
```

Comment:
ChangeLogs should not have cspell directives in them

Reply:
Thanks for your review! I moved these inline cspell ignores to the cspell.json under a filtered path.